Sync/StorageClient.swift

```diff
@@ -232,18 +232,29 @@ public struct StorageResponse<T> {
     }
 }
 
+public typealias BatchToken = Int64
```

Comment:
As noted on an earlier diff: the batch token is an opaque identifier, and is delivered as a JSON string, according to the docs.